src/palette/elements/Select/Select.tsx

```diff
@@ -4,9 +4,8 @@ import { CloseIcon, Flex, Separator, Spacer, Text, Touchable, useColor } from "p
 import React, { useEffect, useMemo, useRef, useState } from "react"
 import { FlatList, TextInput, TouchableOpacity } from "react-native"
 import { FancyModal } from "../../../lib/Components/FancyModal/FancyModal"
-import { INPUT_HEIGHT } from "../../../lib/Components/Input/Input"
 import { SearchInput } from "../../../lib/Components/SearchInput"
-
+import { INPUT_HEIGHT } from "../Input"
 export interface SelectOption<ValueType> {
```

Comment:
```suggestion
import { INPUT_HEIGHT } from "../Input"

export interface SelectOption<ValueType> {
```
just some breathing room